packages/components/psammead-sitewide-links/src/index.jsx

```diff
@@ -44,10 +44,16 @@ const StyledParagraph = styled.p`
   padding: ${GEL_SPACING_DBL} 0;
 `;
 
-const SitewideLinks = ({ links, copyrightText, externalLink, service }) => (
+const SitewideLinks = ({
+  links,
+  trustProjectLink,
```

Comment:
Instead of using a `null` value for default prop below.

```suggestion
  trustProjectLink = null,
```

Reply:
This causes eslint issues.